core/src/com/unciv/ui/screens/civilopediascreen/FormattedLine.kt

```diff
@@ -316,6 +296,28 @@ class FormattedLine (
         return table
     }
 
+    private fun renderExtraImage(labelWidth: Float): Table {
+        val table = Table(BaseScreen.skin)
+        fun getExtraImage(): Image {
+            if (ImageGetter.imageExists(extraImage))
+                return if (centered) ImageGetter.getDrawable(extraImage).cropToContent()
+                    else ImageGetter.getImage(extraImage)
+            val externalImage = ImageGetter.findExternalImage(extraImage)
```

Comment:
This exception is not caught anywhere - which means bad mod definition can cause Unciv crash, 

Instead I think this function should return an empty Table or perhaps null

Reply:
There's a difference between *catching specific scenarios with if checks* and just throwing a try/catch all
In the first scenario, you know what you're dealing with. In the second, you have no clue. 
Unciv should never crash sure to misconfigured mods